apps/user-office-backend/src/models/questionTypes/QuestionRegistry.ts

```diff
@@ -137,3 +143,18 @@ export function getDefaultAnswerValue(
 
   return definition.getDefaultAnswer(questionTemplateRelation);
 }
+
+export async function getExternalApiCallData(record: DynamicRecord) {
+  const definition = getQuestionDefinition(record.data_type as DataType);
+
+  if (!definition.externalApiCall) {
+    logger.logError('Tried to make non-existing definition callback', {
+      dataType: record.data_type as DataType,
+    });
+    throw new GraphQLError('Tried to execute non-existing definition callback');
+  }
+
+  const options = await definition.externalApiCall(record as DynamicRecord);
```

Comment:
Do we need this declaration if we are not it  beyond this point unless it is for other reasons.

Reply:
This declaration can be removed, I included it just for better personal understanding of what to expect from the value returned

Reply:
Okay thanks.